enhancements/operator-hub-check.md

```diff
@@ -0,0 +1,103 @@
+
+## Summary
+
+`OperatorHubValidator` checks of the operator is respecting the required criteria to be published in the [OperatorHub.io](https://operatorhub.io/). This proposal describe criteria to be added on it. 
+
+Many tools and projects uses `OperatorHubValidator` already. Note that the new checks described by this proposal will be available via SDK:
+
+```sh
+To validate a bundle with a superset of requirements for operatorhub.io specifically:
+   
+  $ operator-sdk bundle validate ./bundle --select-optional name=operatorhub
+``` 
+
+**NOTE** For further information check [here](https://sdk.operatorframework.io/docs/olm-integration/generation/#validation) in the SDK docs and the [here](https://olm.operatorframework.io/docs/tasks/validate-package/#validation) in the OLM docs. Also, the new checks are demanding to address a further requirement proposal to audit all bundles of a provided catalogue. 
+
+## Motivation
+
+- Allow operator developers and automatically jobs check required criteria to publish an operator bundle on OperatorHub before it is achieved locally and in the CI 
+- Validate the operator bundle manifests and notify users and maintainers with warning and errors about what is not complain about the OperatorHub criteria
+
+### Goals
+
+- Perform static checks on the operator bundle manifests to avoid the scenarios where the operator bundle:
+    - is published but will not be installable
+    - is not prepared to work well on all Kubernetes cluster versions and has not any configuration to address its limitation
+    - is using deprecated or no longer supported K8S APIs
+    - is not following the good practices and conventions defined 
+- [operator-framework/api][oper-api] with new checks in `OperatorHubValidator`
+- SDK tool using [operator-framework/api][oper-api] and return errors or warnings regarding the new checks via `operator-sdk bundle validate ./bundle --select-optional name=operatorhub`
+- Pipeline performing the new checks since it uses `operator-sdk bundle validate ./bundle --select-optional name=operatorhub` 
+
+### Non-Goals
+
+- Add checks which are specific for some kind of vendor
+
+## Proposal
+
+### User Stories 
+
+- As an operator developer, I'd like to use the SDK tool to validate my operator bundle, which will be published on OperatorHub so that, I could quickly get its errors and warnings locally to be able to provide the required fixes and improvements before try to publish it 
+- As an operator framework maintainer, I'd like to see the `OperatorHubValidator` in [operator-framework/api][oper-api] performing the proposed new checks so that, I can easily keep it maintained and use it in any project and tool which please me.
+- As an operator framework maintainer, I'd like to see CI checking the operator bundle criteria when a user pushes a Pull Request to add the operator bundle on the OpertorHub so that, we can easily ensure a better quality of the projects before they are published. 
```

Comment:
```suggestion
- As an operator framework maintainer, I'd like to see CI checking the operator bundle criteria when a user pushes a Pull Request to add the operator bundle on the OperatorHub so that, we can easily ensure a better quality of the projects before they are published. 
```